rfcs/006 - Replicate changes between staging and prod RFC.md

```diff
@@ -39,7 +39,7 @@ Important requirements to keep in mind are being able to:
 
 There are too many edge cases around our data model and different kinds of staging->production arrangements companies might have (differences in capacity and secrets are a given, plus the two environments could be in different regions altogether) to try to reliably perform an import from one database to another automatically. The alternatives considering automatic workflows would fail or require manual intervention the majority of the time. The expectation for the workflow we set forth should then be that the user will still need to provide a decent amount of manual input.
 
-The IaSQL audit log table is timestamped and looking at the records from a given point in time for a staging database will give users most of the information they need to be able to replicate the changes in production. This proposal is grounded on that fact and seeks to make that workflow as smooth as possible by creating a feature that generates SQL from the audit log from a given point in time. The given point in time for this use case would be the last time staging and production were synced. The generated SQL will then be massaged and manually run by the user in production via a PostgreSQL client or a migration within a plain SQL migration system for teams that want to code review infrastructure changes. The IaSQL audit log is ordered chronologically and the SQL generated by a `iasql_get_sql_since()` function should not cause duplicate records or FK relation problems even if the expectation is that the generated SQL should be massaged.
+The IaSQL audit log table is timestamped and looking at the records from a given point in time for a staging database will give users most of the information they need to be able to replicate the changes in production. This proposal is grounded on that fact and seeks to make that workflow as smooth as possible by creating a feature that generates SQL from the audit log from a given point in time. The given point in time for this use case would be the last time staging and production were synced. The generated SQL will then be massaged and manually run by the user in production. The IaSQL audit log is ordered chronologically and the SQL generated by a `iasql_get_sql_since()` function should not cause duplicate records or FK relation problems even if the expectation is that the generated SQL should be massaged.
```

Comment:
I don't know if we should edit past RFCs like this even if we have changed our perspective in the meantime.

Reply:
I am just removing a detail, more than changing something fundamental about the RFC, since we do not recommend using a migration system and the RFC is reference in https://iasql.com/docs/envs. Seems inconsequential, but I can add a superseded note or something

Reply:
discussed offline. we will remove the mention in the docs of the RFC and keep RFC 006 as is keeping the RFC as historical record of our thought processes behind things